functional-test/src/test/groovy/org/rundeck/util/common/execution/ExecutionUtils.groovy

```diff
@@ -40,6 +40,18 @@ class ExecutionUtils {
             }
         }
 
+        /**
+         * Returns a boolean that verifies if the executions where executed sequentially,
```

Comment:
nit:
In javadoc, you'd specify the return type in the `@return. Also, the best practices prescribe avoiding implementation details (e.g. start-date, end-date) and favour buinsiess meaning.
E.g.
```
Checks if executions were executed sequentially, meaning the execution at a position `n` has ended before the execution at position `n+1` has started.
```